tools/eval_net.py

```diff
@@ -348,7 +353,7 @@ def eval(cfg, args, model, weights_basename, criterion, best_value):
             dataset_name,
             as_test=True,
             add_noise=noise_level > 0,
-            add_motion=include_motion != "false",
+            add_motion=(include_motion != "false" and data_transform is not None),
```

Comment:
`include_motion` is a boolean - you dont need to compare it to false.

it should be

```python
add_motion=include_motion and data_transform is not None,
```